refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/test/java/de/muenchen/refarch/integration/dms/adapter/out/fabasoft/FabasoftAdapterTest.java

```diff
@@ -54,189 +55,197 @@
-        response.setObjid("documentCOO");
+        final UpdateIncomingGIResponse response = new UpdateIncomingGIResponse();
+        response.setObjid(DOCUMENT_COO);
 
         WiremockWsdlUtility.stubOperation(
                 "UpdateIncomingGI",
                 UpdateIncomingGI.class, (u) -> true,
                 response);
 
-        fabasoftAdapter.updateDocument("documentCOO", DocumentType.EINGEHEND, List.of(content), "user");
+        fabasoftAdapter.updateDocument(DOCUMENT_COO, DocumentType.EINGEHEND, List.of(content), USER);
     }
 
     @Test
-    void execute_updateOutgoingDocument_request() throws DmsException {
-        Content content = new Content("extension", "name", "content".getBytes());
+    void executeUpdateOutgoingDocumentRequest() throws DmsException {
+        final Content content = new Content(EXTENSION, NAME, CONTENT);
 
-        val response = new UpdateOutgoingGIResponse();
-        response.setObjid("documentCOO");
+        final UpdateOutgoingGIResponse response = new UpdateOutgoingGIResponse();
+        response.setObjid(DOCUMENT_COO);
 
         WiremockWsdlUtility.stubOperation(
                 "UpdateOutgoingGI",
                 UpdateOutgoingGI.class, (u) -> true,
                 response);
 
-        fabasoftAdapter.updateDocument("documentCOO", DocumentType.AUSGEHEND, List.of(content), "user");
+        fabasoftAdapter.updateDocument(DOCUMENT_COO, DocumentType.AUSGEHEND, List.of(content), USER);
     }
 
     @Test
-    void execute_updateInternalDocument_request() throws DmsException {
-        Content content = new Content("extension", "name", "content".getBytes());
+    void executeUpdateInternalDocumentRequest() throws DmsException {
+        final Content content = new Content(EXTENSION, NAME, CONTENT);
 
-        val response = new UpdateInternalGIResponse();
-        response.setObjid("documentCOO");
+        final UpdateInternalGIResponse response = new UpdateInternalGIResponse();
+        response.setObjid(DOCUMENT_COO);
 
         WiremockWsdlUtility.stubOperation(
                 "UpdateInternalGI",
                 UpdateInternalGI.class, (u) -> true,
                 response);
 
-        fabasoftAdapter.updateDocument("documentCOO", DocumentType.INTERN, List.of(content), "user");
+        fabasoftAdapter.updateDocument(DOCUMENT_COO, DocumentType.INTERN, List.of(content), USER);
     }
 
     @Test
-    void execute_cancelObject_request() throws DmsException {
-        val response = new CancelObjectGIResponse();
+    void executeCancelObjectRequest() throws DmsException {
+        final CancelObjectGIResponse response = new CancelObjectGIResponse();
         response.setStatus(0);
 
         WiremockWsdlUtility.stubOperation(
                 "CancelObjectGI",
                 CancelObjectGI.class, (u) -> true,
                 response);
 
-        fabasoftAdapter.cancelObject("objectCoo", "user");
+        fabasoftAdapter.cancelObject("objectCoo", USER);
     }
```

Comment:
Can we add some kind of assertion to test the successful case? Won't comment on any further occurences from now on... 😄